domains/itsvaibhav.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Vaibhav1337",
+        "email": "vaibhavtripathi1337@gmail.com"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=itsvaibhav.is-a.dev,8b5014bc53898eae14fb"
```

Comment:
vercel is not supported yet (might be soon)

Reply:
is netlify supported?, if not can u pls suggest few of them